go.mod

```diff
@@ -4,29 +4,30 @@ go 1.12
 
 require (
 	github.com/NYTimes/gziphandler v0.0.0-20170623195520-56545f4a5d46
-	github.com/PuerkitoBio/purell v1.0.0 // indirect
-	github.com/PuerkitoBio/urlesc v0.0.0-20160726150825-5bd2802263f2 // indirect
-	github.com/davecgh/go-spew v1.1.0
+	github.com/asaskevich/govalidator v0.0.0-20190424111038-f61b66f89f4a
+	github.com/davecgh/go-spew v1.1.1
 	github.com/emicklei/go-restful v0.0.0-20170410110728-ff4f55a20633
 	github.com/ghodss/yaml v0.0.0-20150909031657-73d445a93680
-	github.com/go-openapi/jsonpointer v0.0.0-20160704185906-46af16f9f7b1 // indirect
-	github.com/go-openapi/jsonreference v0.0.0-20160704190145-13c6e3589ad9 // indirect
-	github.com/go-openapi/spec v0.0.0-20160808142527-6aced65f8501
-	github.com/go-openapi/swag v0.0.0-20160704191624-1d0bd113de87 // indirect
+	github.com/go-openapi/jsonpointer v0.19.3
+	github.com/go-openapi/jsonreference v0.19.4
+	github.com/go-openapi/spec v0.19.12
+	github.com/go-openapi/swag v0.19.11
 	github.com/golang/protobuf v1.3.3
 	github.com/google/go-cmp v0.4.0 // indirect
 	github.com/google/gofuzz v1.0.0
+	github.com/google/uuid v1.1.1
 	github.com/googleapis/gnostic v0.4.1
 	github.com/json-iterator/go v1.1.6
-	github.com/mailru/easyjson v0.0.0-20160728113105-d5b7844b561a // indirect
+	github.com/kr/text v0.2.0 // indirect
```

Comment:
k/k uses `v0.1.0` but I don't think it should matter looking at https://github.com/kr/text/compare/v0.1.0...v0.2.0 and https://github.com/creack/pty/compare/v1.1.7...v1.1.9.